jina/drivers/convert.py

```diff
@@ -8,6 +8,7 @@
 import urllib.request
 import zlib
 
+from PIL import Image
```

Comment:
Better move this import inside the function needing it

Reply:
I have to keep it outside for argument type info for image_to_byte_array. Should I remove type hint from the argument of image_to_byte_array? 

I have kept PIL inside the function in png_convertor3d

Reply:
if using for type hint you can do:

```python
if False:
   from PIL import Image
```